dev/Dockerfile-dev

```diff
@@ -23,7 +23,9 @@ RUN chmod 777 /root
 
 EXPOSE 9002
 EXPOSE 9003
+EXPOSE 5678
 
 ENV OPENSLIDES_DEVELOPMENT 1
 
-CMD [ "python", "-m", "openslides_backend" ]
+# CMD [ "python", "-m", "openslides_backend" ]
+CMD python -m debugpy --listen 0.0.0.0:5678 openslides_backend
```

Comment:
Use the `exec` form for CMD: https://docs.docker.com/engine/reference/builder/#cmd

Reply:
Comment line deleted.
Please, let me stay with the shell-style CMD. It's the only one I got running. (My) problem is the debugpy-module and it's listen parameter.

Reply:
I've opened https://github.com/OpenSlides/openslides-backend/issues/475 for it - I'll have a look.